examples/bindings/components/bindings.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: binding-example
+spec:
+  type: bindings.azure.servicebusqueues
```

Comment:
Most of the Dapr examples rely on the components that are locally installed with `dapr init`, meaning Redis. Can you change the pub/sub component to use Redis?

Reply:
@yaron2 I don't get any default components installed with `dapr init --slim`.
Should the component just be removed from the example (the other examples don't have one)?
Or, should it just be copy/pasted from here: https://docs.dapr.io/reference/components-reference/supported-bindings/redis/

Reply:
@NickLarsenNZ - The examples are usually based off `dapr init` or `dapr init -k --dev`
A good starting point would be here: https://github.com/dapr/go-sdk/tree/main/examples/service/config
I'm looking to get all the examples validating as part of a new github actions workflow using `dapr/mechanical-markdown`